docs/portal/image-registry/editing-projects.md

```diff
@@ -0,0 +1,16 @@
+# Editing Projects
```

Comment:
1. "Editing" is a concept similar to "Updating". Having both "Edting Projects" page and "Updating Projects" page would probably confuse readers. In previous documentation, you've used "Viewing" to distinguish from "Updating". Please do the same here. 
2. The contents below are a bit inconsistent with previous pages. Please refer to this page "viewing-image-pull-secret-information" as an example and try to maintain the consistency.

Note: the above 2 points apply to "editing users" page as well.

In addition, please take out the contents on repository and tags, and put them under 2 new separate pages, i.e. "Managing Repositories Under a Project" and "Managing Tags Under a Repository". The first page will explain how to view repositories under a project, how to update a repository (i.e. change description) and how to delete a reposiroty. The second page will explain how to view tags under a repository, how to view the details of a tag, and how to delete tags. FYI, the relationship among project, repository, and tag is that a project can have one or multiple repositories, and that a repository can have one ore multiple tags. We'd better reorganize the contents to make the relationship clearer.

Reply:
Done.